torch/_decomp/decompositions.py

```diff
@@ -1244,10 +1244,10 @@ def transpose_int(self: Tensor, dim0: int, dim1: int) -> Tensor:
     dim0, dim1 = utils.canonicalize_dims(self.dim(), (dim0, dim1))  # type: ignore[misc]
 
     if self.dim() <= 1:
-        return self
+        return self.view(self.shape)
```

Comment:
why does this become a view now?

Reply:
This is a hack to guarantee we return a fresh tensor; returning an input in a view is gonna give you a bad time

Reply:
gotcha, do you want to add a comment for that?